src/ShapeSigAng.cc

```diff
@@ -143,37 +241,65 @@ Double_t ShapeSigAng::analyticalIntegral(Int_t code, const char* rangeName) cons
 
   Double_t ret;
 
+  double limitFac = 0;
+  if (useSWave) limitFac = interferenceFactor * sqrt( 3. * Fs * (1-Fs) );
```

Comment:
sorry I don't remember how this is derived

Reply:
The parameters used in the fit are not the real "A" parameters, but rescaled one, so that their range is [-1,1]. This happens by replacing in the PDF each term, like
As*sin^2(thetaL)*cos(thetaK),
with the term explicitly containing the parameter's limit and a new parameter, like
A'_s*F_theo*2*sqrt(3*Fl*Fs*(1-Fs))*sin^2(thetaL)*cos(thetaK),
where the limit is obtained from Eq. 50 in [1303.5794](https://arxiv.org/pdf/1303.5794.pdf).
Since the factor F_theo*sqrt( 3. * Fs * (1-Fs) ) is common in all the limits, its value has been assigned to a variable to keep the formula more compact. The default value of F_theo is taken to be 0.9, as mentioned in the article for K* mass windows of 100 MeV (since the K* mass is used as BDT variable and there is no clear window, this is going to be an approximation)